misc/helm-charts/operator/README.md

```diff
@@ -91,17 +91,7 @@ To enable observability features, set `observability.enabled=true`. This will cr
 
 ### Network Policies
 
-Network policies can be enabled by setting `networkPolicies.enabled=true`.
-
-To enable policies for internal communication between Materialize pods, set `networkPolicies.internal.enabled=true`.
-
-To enable policies for ingress HTTP or SQL communication to Materialize environmentd or balancerd pods, set `networkPolicies.ingress.enabled=true`.
-
-Ingress policies can configure the list of CIDR blocks to allow with `networkPolicies.ingress.cidrs`.
-
-To enable policies for egress communication from Materialize pods to sources and sinks, set `networkPolicies.egress.enabled=true`.
-
-Egress policies can configure the list of CIDR blocks to allow with `networkPolicies.ingress.cidrs`.
+Network policies can be enabled by setting `networkPolicies.enabled=true`. By default, the chart uses native Kubernetes network policies. To use Cilium network policies instead, set `networkPolicies.useNativeKubernetesPolicy=false`.
```

Comment:
Is this an auto-generated file? This seems to be reverting and changing some things in a bad way.

Reply:
yeah, it's generated via `helm-docs`, although i'm not quite sure how it works

Reply:
Yes, it is auto-generated from the README.md.tmpl file. The `helm-docs` uses the tmpl file as the base, and then scans the chart and generate the table with the resources. We should idealy not make manual changes to the README.md itself, but the README.md.tmpl file and then run the `helm-docs` to generate the README.md file.